docs/data-cookbooks/setup.md

```diff
@@ -0,0 +1,103 @@
+
+**Break down of [s3_provider.json](https://github.com/nasa/cumulus/tree/tree/example/data/providers/s3_provider.json):**
+
+|Key|Value|Required|Description|
+|:---:|:-----:|:------:|-----------|
+|id|`"s3_provider"`|Yes|Unique identifier for provider|
+|globalConnectionLimit|`10`|Yes|Integer specifying the connection limit to the provider|
+|protocol|`"s3"`|Yes|(http\|https\|ftp\|sftp\|s3) are current valid entries|
+|host|`"cumulus-data-shared"`|Yes|Host where the files will exist or s3 bucket if "s3" provider|
+|port|`${port_number}`|No|Port to connect with the provider on|
+|username|`${username}`|No|Username for access to the provider. Plain-text or encrypted. Encrypted is highly suggested|
+|password|`${password}`|No|Password for accces to the provider. Plain-text or encrypted. Encrypted is highly suggested|
+
+_The above optional attributes are not shown in the example provided, but they have been included in this document for completeness_
+
+
+### Rules
+
+Rules are used by to start processing workflows and the transformation process. Rules can be invoked manually, based on a schedule, or can be configured to be triggered by events in [Kinesis](./cnm-workflow.md) or SNS. The current best way to understand rules is to take a look at the [schema](https://github.com/nasa/cumulus/tree/master/packages/api/models/schemas.js) (specifically the object assigned to `module.exports.rule`). Rules can be viewed, edited, added, and removed from the Cumulus dashboard under the "Rules" navigation tab. Additionally, they can be managed via the [rules api](https://nasa.github.io/cumulus-api/?language=Python#list-rules).
+
+We don't currently have examples of rules in the Cumulus repo, but we can see how to create a rule from the Cumulus dashboard.
+1. In the Cumulus dashboard, click `Rules` on the navigation bar.
+2. Click the `Add a rule` button.
+
+|Key|Value|Required|Description|
+|---|-----|--------|-----------|
+|name|`"myScheduledRule"`|Yes|Name of the rule. This is the name under which the rule will be listed on the dashboard|
+|workflow Name|`"ParsePdr"`|Yes|Name of the workflow to be run. A list of available workflows can be found on the Workflows page|
+|Provider Id|`"myProvider"`|Yes|Configured provider's iD. This can be found on the Providerse page|
+|Collection Name|`"myCollection"`|Yes|Name of the collection this rule will moderate. Configured and found in the Collections page|
+|Collection Version|`"006"`|Yes|Version of the collection this rule will moderate. Configured and found in the Collections page|
+|Rule - Type|`onetime`|Yes|(onetime\|scheduled\|sns\|kinesis) type of scheduling/workflow kick-off desired|
+|Rule - Value|[here](#rule-value)|Yes|This entry depends on the type of run|
+|Rule state|`ENABLED`|Yes|(ENABLED\|DISABLED) whether or not the rule will be active|
+|Optional tags|`"nightly"`|No|A string type tag that can be added to simplify search|
+
+#### rule-value
+```
```

Comment:
If this is just text, this doesn't need to be in a code block. Also you could put the scheduled valid values in with this instead of referring down to the bottom.

Reply:
good call :+1: